src/foam/nanos/medusa/MedusaAdapterDAO.js

```diff
@@ -96,6 +146,32 @@ It then marshalls it to the primary mediator, and waits on a response.`,
   ],
 
   methods: [
+    {
+      name: 'init_',
+      javaCode: `
+      getIndexStateDAO();
+      `
+    },
+    {
+      name: 'find_',
+      javaCode: `
+      DAO dao = getDelegate();
```

Comment:
Should this block of code be made into a helper method, since it appears in more than one spot?
Is this thread-safe?

Reply:
Maybe name it something like getReadDelegate().